civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -0,0 +1,63 @@
+/* Page: My details */
+
+.page-hr-details {
+  .civihr-employee-portal-my-details-block {
+    background: $light;
+    padding: 20px;
+    margin: 0 15px;
```

Comment:
There are a lot of places in this change that mention 15px, I wonder if they are connected. If they are, then 15px should go to variables. Moreover, I think the name of the variable should *not* be something like "block-margin", but rather smth like "block-gap" so this var can be reused in both margins, paddings etc.

Reply:
I looked at all the scss from SSP theme, looks like we have a mixture of 15px and 20px all over the place. 
So 
1. Creating a single variable would not solve this problem as we have different values
2. We could try to use a single value like either 15 or 20, but then the change will be huge and scope will increase a lot. And this ticket needs to be done by today.

Reply:
OK, but please consider refactoring it when there is a chance since having these values all over the place can be an issue for a further development.